src/main/java/org/casbin/jcasbin/model/Policy.java

```diff
@@ -187,6 +190,16 @@ public boolean addPolicy(String sec, String ptype, List<String> rule) {
             List<List<String>> policy = assertion.policy;
             int priorityIndex = assertion.priorityIndex;
 
+            String line = ptype + "," + String.join(",", rule);
```

Comment:
We have several ways to get new policy imported to Casbin, including loadPolicy, addPolicy, updatePolicy. Even removePolicy needs to use existing policy rule as parameter. Will the others need this handling too? You should put your code into a more critical code path like a util to handle all rule parsing, not only addPolicy inside.